galpy/orbit/integrateFullOrbit.py

```diff
@@ -300,6 +300,8 @@ def _parse_pot(pot,potforactions=False,potfortorus=False):
             pot_args.extend([p._amp])
             pot_args.extend([0.,0.,0.,0.,0.,0.]) # for caching
             pot_args.extend(list(p._rot.flatten()))
+            pot_args.append(not p._offset is None)
+            pot_args.extend(list(p._offset))
```

Comment:
I think because of my other suggestions, this now has to be

```suggestion
            pot_args.extend(list(p._offset) if not p._offset is None else [0.,0.,0.])
```

which I think should be valid Python? Probably want to check...

Reply:
included!